server/bin/pbench-verify-backup-tarballs.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, name, dirname, tmpdir, logger):
             self.dirname = None
         else:
             if not os.path.isdir(dirname):
-                raise Exception("Bad {}: {}".format(name, dirname))
+                raise Exception("Bad {}: {}", name, dirname)
```

Comment:
Since this is not logging, just creating an `Exception()` object, it would be good to use f-strings in this case.

Reply:
my bad. Thanks